From 49cd9ea6dc8d68eb519ccd9f31c9730dec8a181a Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Thu, 8 Mar 2018 22:14:50 +0100
Subject: [PATCH] Revert "ARM: dts: sun7i: Add BCM53125 switch nodes to the
 lamobo-r1 board"

This reverts the changes needed for the upstream b53 DSA switch driver 
to use the OpenWrt b43 swconfig switch driver.

This reverts commit 0cdefd5b5485ee6eb3512a75739d09a4090176ed.
This reverts commit d7b9eaff5f0ca00726336b4c0c3c29decf30412a.
---
 arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts | 60 ++-----------------------------
 1 file changed, 3 insertions(+), 57 deletions(-)

--- a/arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts
+++ b/arch/arm/boot/dts/sun7i-a20-lamobo-r1.dts
@@ -124,67 +124,13 @@
 &gmac {
 	pinctrl-names = "default";
 	pinctrl-0 = <&gmac_pins_rgmii_a>;
+	phy = <&phy1>;
 	phy-mode = "rgmii";
 	phy-supply = <&reg_gmac_3v3>;
 	status = "okay";
 
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
-
-	mdio {
-		compatible = "snps,dwmac-mdio";
-		#address-cells = <1>;
-		#size-cells = <0>;
-
-		switch: ethernet-switch@1e {
-			compatible = "brcm,bcm53125";
-			reg = <30>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-
-			ports {
-				#address-cells = <1>;
-				#size-cells = <0>;
-
-				port0: port@0 {
-					reg = <0>;
-					label = "lan2";
-				};
-
-				port1: port@1 {
-					reg = <1>;
-					label = "lan3";
-				};
-
-				port2: port@2 {
-					reg = <2>;
-					label = "lan4";
-				};
-
-				port3: port@3 {
-					reg = <3>;
-					label = "wan";
-				};
-
-				port4: port@4 {
-					reg = <4>;
-					label = "lan1";
-				};
-
-				port8: port@8 {
-					reg = <8>;
-					label = "cpu";
-					ethernet = <&gmac>;
-					phy-mode = "rgmii-txid";
-					fixed-link {
-						speed = <1000>;
-						full-duplex;
-					};
-				};
-			};
-		};
+	phy1: ethernet-phy@1 {
+		reg = <1>;
 	};
 };
 
